From 22c5a278b98cf6ea4e4a402bca19d1354ab3c431 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Tue, 25 Sep 2018 16:05:17 +0300
Subject: [PATCH] Disable build for tools and use mkimage from sysroot

fails when trying to compile with host gcc: unable to execute
'x86_64-linux-gnu-gcc': No such file or directory
---
 Makefile             | 2 +-
 scripts/Makefile.lib | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 119ad49eec..022690e26e 100644
--- a/Makefile
+++ b/Makefile
@@ -698,7 +698,7 @@ libs-y += $(if $(BOARDDIR),board/$(BOARDDIR)/)
 
 libs-y := $(sort $(libs-y))
 
-u-boot-dirs	:= $(patsubst %/,%,$(filter %/, $(libs-y))) tools examples
+u-boot-dirs	:= $(patsubst %/,%,$(filter %/, $(libs-y)))
 
 u-boot-alldirs	:= $(sort $(u-boot-dirs) $(patsubst %/,%,$(filter %/, $(libs-))))
 
diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index 80ddb08474..e86c949f08 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -493,5 +493,5 @@ cmd_xzmisc = (cat $(filter-out FORCE,$^) | \
 # ---------------------------------------------------------------------------
 MKIMAGEOUTPUT ?= /dev/null
 quiet_cmd_mkimage = MKIMAGE $@
-cmd_mkimage = $(objtree)/tools/mkimage $(MKIMAGEFLAGS_$(@F)) -d $< $@ \
+cmd_mkimage = mkimage $(MKIMAGEFLAGS_$(@F)) -d $< $@ \
 	$(if $(KBUILD_VERBOSE:1=), >$(MKIMAGEOUTPUT))
